main.go

```diff
@@ -54,9 +74,9 @@ func main() {
 	// object content
 	e.GET("/object/metadata", auth.Authorize(bh.HandleGetMetaData, allUsers...))
 	e.GET("/object/content", auth.Authorize(bh.HandleObjectContents, allUsers...))
-	e.PUT("/object/move", auth.Authorize(bh.HandleMoveObject, writer...))
+	e.PUT("/object/move", auth.Authorize(bh.HandleMoveObject, admin...))
```

Comment:
I changed to admin since we are not providing FGAC here (@slawler requested) and `move` involves deletion, hence it should be scoped to admin.
I think other `move` should also be scoped to admins only.

Reply:
Gotcha, I chatted with Seth. Please change the other move to admin as well. Thanks!

Reply:
Done